tests/test_backend_loki_fields.py

```diff
@@ -0,0 +1,95 @@
+from typing import Any, Dict, Tuple
+import pytest
+from sigma.backends.loki import LogQLBackend
+from sigma.collection import SigmaCollection
+from sigma.exceptions import SigmaFeatureNotSupportedByBackendError, SigmaTypeError
+from sigma.modifiers import modifier_mapping
+
+
+@pytest.fixture
+def loki_backend() -> LogQLBackend:
+    return LogQLBackend()
+
+
+# Mapping from modifier identifier strings to modifier classes
+modifier_sample_data: Dict[str, Tuple[Any, str]] = {
+    # "modifer": (value, expected_output)
+    "contains": ("valueA", "fieldA=~`(?i).*valueA.*`"),
+    "startswith": ("valueA", "fieldA=~`(?i)valueA.*`"),
+    "endswith": ("valueA", "fieldA=~`(?i).*valueA`"),
+    "exists": ("yes", 'fieldA!=""'),
+    "base64": ("valueA", "fieldA=~`(?i)dmFsdWVB`"),
+    "base64offset": (
+        "valueA",
+        "fieldA=~`(?i)dmFsdWVB` or fieldA=~`(?i)ZhbHVlQ` or fieldA=~`(?i)2YWx1ZU`",
+    ),
+    "wide": ("valueA", "fieldA=~`(?i)v\x00a\x00l\x00u\x00e\x00A\x00`"),
+    "windash": ("-foo", "fieldA=~`(?i)\\-foo` or fieldA=~`(?i)/foo`"),
+    "re": (".*valueA$", "fieldA=~`.*valueA$`"),
+    "i": ("valueA", "---"),
+    "ignorecase": ("valueA", "---"),
+    "m": (["valueA", "valueB"], "---"),
```

Comment:
Yes, I don't think we can meaningfully support multiline queries as LogQL does not seem to have any support for it (the recommended approach for handling such logs is to [convert them into a single line](https://grafana.com/docs/loki/latest/send-data/promtail/stages/multiline/) with a special delimiter in the ingest phase, like a zero-width space). Worth adding a comment to reflect this fact?

Reply:
Addressed in https://github.com/grafana/pySigma-backend-loki/pull/151/commits/ebd93a851d9d123a7a1f2fa8243294ff9b88bc3d.